docs/misc/kafka-pv-hostpath.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
you need to add a similar manifest for zookeeper

Reply:
ok